Logger/BaseLoggerMixins.cs

```diff
@@ -1,6 +1,28 @@
-﻿namespace Logger;
+﻿using System;
+namespace Logger;
+
+    public static class BaseLoggerMixins
+    {
+        public static void Error(this BaseLogger baseLogger, string message, params object[] args) 
```

Comment:
```suggestion
        public static void Error(this BaseLogger logger, string message, params object[] args) 
```
Name this logger, instead of baseLogger, because this could be a typeof BaseLogger (e.g. a FileLogger), so the variable name is misleading here.